apps/@sparrow-web/src/pages/workspace-page/CollectionsPage.svelte

```diff
@@ -72,11 +73,13 @@
   import { SocketIORequestDefaultAliasBaseEnum } from "@sparrow/common/types/workspace/socket-io-request-base";
   import GraphqlExplorerPage from "./sub-pages/GraphqlExplorerPage/GraphqlExplorerPage.svelte";
   import { GraphqlRequestDefaultAliasBaseEnum } from "@sparrow/common/types/workspace/graphql-request-base";
+  import { TeamsViewModel } from "../Teams/Teams.ViewModel";
```

Comment:
@aakashreddy-p  don't import teams view model here, in MVVM pattern we only import the view model with respective to it's page, so whatever functions you require add them in collectionview model and then use them.